apps/transport/lib/transport_web/templates/resource/_resources_details.html.heex

```diff
@@ -17,16 +17,33 @@
           <strong><%= Enum.join(@modes, ", ") %></strong>
         </li>
       <% end %>
-      <%= if Map.get(@metadata, "networks", []) != [] do %>
+      <% networks_start_end_dates = Map.get(@metadata, "networks_start_end_dates", %{}) %>
+      <% networks = Map.get(@metadata, "networks", []) %>
+      <%= if Enum.count(networks) > 1 && networks_start_end_dates != %{} do %>
```

Comment:
On utilise pas plus `and` ?

Reply:
on aurait pu, `and` est plus stricte que `&&`